### DISCUSSION TITLE
How to override m2e global settings?

### DISCUSSION THREAD

**Discussion by skyhirider** [eclipse-m2e/m2e-core#1289] [Q&A]
I would like to define global settings.xml on a shared multi-user Windows system, but was unable to find how.

By default, Eclipse is looing into %userprofile%/.m2/settings.xml or user.home, but I have not found a way to override it to without copying the settings into each individual user profile, which I do not want to do as that would make deploying global changes problematic.

The m2e documentation states that "maven home" can be used but I was not able to get it working - https://www.eclipse.org/m2e/documentation/m2e-faq.html#how-to-configure-proxy-and-location-of-maven-local-repository

Which path variable can I set on Windows to force Eclipse to read maven settings from there by default?

**Comment by laeubi** [comment]
![grafik](https://user-images.githubusercontent.com/1331477/222502303-65bede1c-3dff-4c3d-a416-f43e42c23ca5.png)

**Comment by skyhirider** [comment]
Thanks, but this is set per project/user, I need it set up once for all
users that log in to a shared windows environment.

**Reply by laeubi** [reply]
This setting is per eclipse workspace, if you need to automate this you can use Oompf, I'm not aware of any "windows wide" way.
